content/applications/hr/time_off.rst

```diff
@@ -0,0 +1,625 @@
+presents a list view of all the time off requests for the employee.
+
+To create a new request for time off, click either the :guilabel:`New Time Off` button on the main
+*Time Off* dashboard, or the :guilabel:`Create` button in the :guilabel:`My Time Off` list view.
+Both buttons open a new time off request form. Enter the following information on the form:
+
+- :guilabel:`Time Off Type`: select the type of time off being requested from the drop-down menu.
+- :guilabel:`Dates`: enter the dates that the time off will fall under. There are two fields to
+  populate, the :guilabel:`From` and :guilabel:`To` fields. Click on either the :guilabel:`From` or
+  :guilabel:`To` field, and a calendar pop-up appears. Click on the start date, then click on the
+  end date. The selected start and end dates appear in deep purple, and the dates between them
+  appear in pale purple (if applicable). If the time off requested is for a single day, click on the
+  start date, then click the same date again for the end date. When the correct dates are
+  selected/highlighted, click the :guilabel:`Apply` button. The dates now populate the
+  :guilabel:`From` and :guilabel:`To` fields.
+
+  - :guilabel:`Half Day`: if the time off request is for a half day, activate this box. When this is
+    selected, the :guilabel:`From` date field disappears, and is replaced with a drop-down menu.
+    Select either :guilabel:`Morning` or :guilabel:`Afternoon` to indicate which half of the day is
+    requested.
+  - :guilabel:`Custom Hours`: if the time off requested is not a whole or half day, activate this
+    box. A :guilabel:`From` and :guilabel:`To` field appears beneath this option if selected. Using
+    the drop-down menu, select the start and end time for the time off request.
+
+- :guilabel:`Duration`: this field updates automatically once the :guilabel:`Date` section is
+  completed. If the :guilabel:`Date` section is modified, this section automatically updates to
+  reflect the total time off requested. This field is in either hours or days, depending on the
+  :guilabel:`Date` selections.
+- :guilabel:`Description`: enter a description for the time off request. This should include any
+  details that the manager and approvers need in order to approve the request.
+- :guilabel:`Supporting Document`: if the :guilabel:`Time Off Type` selected allows for the
+  attachments of documents, this field appears. Click the :guilabel:`Attach File` button and a file
+  explorer window appears. Navigate to the file(s) to attach, then click the :guilabel:`Open`
+  button. The files then appear on the time off request form. Multiple documents can be attached.
+
+When the form is complete, click the :guilabel:`Save` button to save the information and submit the
+request.
+
+.. image:: time_off/time-off-request.png
+   :align: center
+   :alt: A time off request form filled out for an employee home sick for two days with the flu.
```

Comment:
No red call-outs on these images

Reply:
Since the entire form is explained above, I did not place any call-outs since it would all be called out.